specs/0107-pretranslated-status.md

```diff
@@ -0,0 +1,48 @@
+- Feature Name: Pretranslated status
+- Created: 2020-09-21
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1666320
+
+# Summary
+
+Introduce a new translation status for Pretranslated strings to differentiate them from Fuzzy strings.
+
+# Motivation
+
+Pretranslation is the process of using machines to translate content before it's translated by human translators. If pretranslation is enabled for the project, any newly added source string gets pretranslated using translation memory (if 100% match is found) or machine translation.
+
+Pretranslations are assigned translation status Fuzzy, which allows us to:
+1. Immediately send them to version control system.
+2. Make them available in the product.
+3. Inform localizers that these strings should be given another look.
+4. Run quality checks on them to identify any errors or warnings.
+
+The caveat however is that translation status Fuzzy is also used for [fuzzy](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html) translations of the Gettext system (filenames ending in .po), which are **not** used in the product. That means bullet point #2 isn't valid for projects using the Gettext system - pretranslated strings in .po files will not make it to the product unless human translator approves them.
+
+We need to make pretranslation work according to all 4 items from the list above, consistently across all file formats (including .po).
+
+# Feature explanation
+
+The obvious solution is to introduce a new "Pretranslated" translation status to differentiate pretranslated strings from Fuzzy. However, while it solves the problem, adding a new translation status also adds more data to the already condensed dashboards and increases complexity.
+
+So instead, we make the following two steps:
+1. Treat Gettext Fuzzy strings as Missing.
```

Comment:
I have some anecdotal data (you probably noticed the same in sl) showing that people tend to ignore the fuzzy strings and only click on "Missing" from the project dashboard. They are probably assuming Missing includes Fuzzy, which would now be the case.
For some time now, I’ve been doing what you’re proposing by batch-rejecting fuzzy suggestions as they land, so that the strings end up in the Missing bucket.

In short, I approve (and welcome) this change